src/Aspire.Hosting/Publishing/ManifestPublishingContext.cs

```diff
@@ -8,24 +8,30 @@
 namespace Aspire.Hosting.Publishing;
 
 /// <summary>
-/// TODO: Doc Comments
+/// Contextual information used for manifest publishing during this execution of the AppHost.
 /// </summary>
-/// <param name="manifestPath"></param>
-/// <param name="writer"></param>
-public sealed class ManifestPublishingContext(string manifestPath, Utf8JsonWriter writer)
+/// <param name="executionContext">Global contextual information for this invocation of the AppHost.</param>
+/// <param name="manifestPath">Manifest path passed in for this invocation of the AppHost.</param>
+/// <param name="writer">JSON writer used to writing the manifest.</param>
+public sealed class ManifestPublishingContext(DistributedApplicationExecutionContext executionContext, string manifestPath, Utf8JsonWriter writer)
 {
     /// <summary>
-    /// TODO: Doc Comments
+    /// Gets execution context for this invocation of the AppHost.
+    /// </summary>
+    public DistributedApplicationExecutionContext ExecutionContext { get; } = executionContext;
```

Comment:
Do we have any uses of this?

Reply:
Yes. As part of manifest publishing we invoke the envvar callback, which needs the executioncontext so those callbacks can do the right thing.